res/css/views/dialogs/_LeaveSpaceDialog.scss

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_LeaveSpaceDialog_wrapper {
+    .mx_Dialog {
+        display: flex;
+        flex-direction: column;
+        padding: 24px 32px;
+    }
+}
+
+.mx_LeaveSpaceDialog {
+    width: 440px;
+    height: 500px;
```

Comment:
this is potentially pretty big for a netbook, do we have any guidelines around supported screen sizes / dialog sizes?

Reply:
This is as specified by the design to maintain a non-jumpy experience when you toggle elements which cause the internal heights to change

We do not have any policy on supported screen sizes, the app does not function well on low resolution screens whatsoever, this doesn't change that. https://github.com/vector-im/element-web/issues/6488